cmd/languageserver.go

```diff
@@ -19,7 +19,7 @@ func init() {
 		Short: "Run the Regal Language Server",
 		Long:  `Start the Regal Language Server and listen on stdin/stdout for client editor messages.`,
 
-		RunE: wrapProfiling(func(args []string) error {
+		RunE: wrapProfiling(func(_args []string) error {
```

Comment:
I'd just do `func([]string)` in this and the similar cases...

Reply:
Maybe it's obvious enough in this case? But I did that first and it just looked like something I'd wonder about 3 months later.

Reply:
Alright, I caved in to your demands :)